apps/polls/assets/QuestionForm.jsx

```diff
@@ -56,10 +56,9 @@ let QuestionForm = React.createClass({
                 htmlFor={'id_questions-' + this.props.key + '-name'}>
                 {django.gettext('Question:')}
               </label>
-              <input
+              <textarea
```

Comment:
Currently the Question.label field is declared as a CharField with a max length of 255. If longer text is required we should change it in the database, too